samples/csharp/MicrosoftGraphSDK/Module.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Graph.Beta;
+using Microsoft.Graph.Beta.Models;
+
+namespace MicrosoftGraphSDK
+{
+    /// <summary>
+    /// Graph SDK endpoints for modules
+    /// </summary>
+    public class Module
+    {
+        public static async Task<EducationModule> CreateAsync(
+            GraphServiceClient client,
+            string classId,
+            string displayName,
+            string description)
```

Comment:
Add header to each method as we have for methods from "Submission.cs" and "Assignment.cs" files.
the header must have a brief description, the parameters that it receives and the return type.

Do it for all methods in this file

Reply:
updated summary